RetroAchievements/RAWeb
# feat: default sort All Games by player count

Behavior analytics is signaling that, of users who visit the title-sorted All Games page (not from the Guest CTA), 10% sort by `-playersTotal`.

This is such a large outlier for this particular page it probably makes sense to change the page's default sort order.

This PR:
* Sorts All Games by `-playersTotal` by default.
* Makes it easier to manage default column visibility, filters, and sort order across all the tables by centralizing this logic into common hooks.

**How to test this PR**
* Visit each of the four tables (all games, system games, backlog games, hub games) with and without a persistence cookie. All Games should sort by player count by default. The other tables should work as normal (tests already should verify this, but no harm in double checking).
